server/common/utils/index.js

```diff
@@ -176,6 +210,38 @@ const responseWithListMembers = (
     isMember: checkIfArrayContainsUserId(memberIds, user._doc._id)
   }));
 
+const responseWithComment = (comment, avatarUrl, displayName) => {
+  const { _id, authorId, createdAt, text } = comment.toObject();
+
+  return {
+    _id,
+    authorAvatarUrl: avatarUrl,
```

Comment:
why are you maping to another name? is this necessary?

Reply:
we store user's Id as authorId in comment and item model, we use authorName to store user's display name in comments and item on frontend, so adhering to the convention I map avatarUrl to authorAvatarUrl...